merlin/dataloader/ops/embeddings.py

```diff
@@ -181,7 +100,7 @@ def compute_output_schema(
         col_schemas.append(
             ColumnSchema(
                 name=self.embedding_name,
-                tags=[Tags.CONTINUOUS, Tags.EMBEDDING],
+                tags=[Tags.EMBEDDING],
                 dtype=self.embeddings.dtype,
                 is_list=True,
                 is_ragged=False,
```

Comment:
We might need to update this `compute_output_schema` to handle list inputs and the ragged case.

Reply:
We don't support ragged sets of embeddings right now, so I think we're okay there. Not sure what you mean re: list inputs—Line 186/105 looks like it _only_ handles the list input case right now, so I suppose it may need an update for the scalar case?

Reply:
Ohhhh, I see what you mean about the ragged case. `is_ragged` should match the id column, right?

Reply:
After talking through what `is_list` and `is_ragged` should be in which cases, we remembered that we should just set the dims/shape and let schemas sort out the values of the flags 😅

Reply:
> We don't support ragged sets of embeddings right now

Looks like it's updated to match the output shape now 😄   It seems this PR is adding support for ragged sets of embeddings. If the ids are ragged you'll get ragged embeddings out

Reply:
By the "list case" I was referring to what we were doing setting the value_count which was only valid if the ids were scalers not lists, since the value_count corresponds to the 2nd dimension